app/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,31 @@
+<resources>
+    <string name="app_name">Weather_3</string>
+    <string name="current_location">Current location</string>
+    <string name="last_update_time">Last updated time</string>
+    <string name="current_temperature">Current temperature</string>>
+    <string name="detail">Detail</string>
+    <string name="humidity_percent">Humidity percent</string>
+    <string name="temperature_unit">Temperature Unit</string>
+    <string name="wind_speed_unit">Wind speed Unit</string>
+    <string-array name="temperature_unit_array">
+        <item>Farenheit</item>
```

Comment:
WARNING: [Typos] "Farenheit" is a common misspelling; did you mean "Fahrenheit" ?
 This check looks through the string definitions, and if it finds any words that look like likely misspellings, they are flagged.